runtime-modules/content/src/tests/transfers.rs

```diff
@@ -232,290 +453,327 @@ fn accept_transfer_status_fails_with_non_channel_owner() {
+            ))
     })
 }
 
 #[test]
-fn accept_transfer_status_succeeds_for_members_to_curators_with_price() {
+fn cancel_channel_transfer_fails_with_invalid_channel_id() {
     with_default_mock_builder(|| {
-        run_to_block(1);
-
-        create_initial_storage_buckets_helper();
-        increase_account_balance_helper(DEFAULT_MEMBER_ACCOUNT_ID, INITIAL_BALANCE);
-        create_default_member_owned_channel();
+        ContentTest::with_member_channel().setup();
+        InitializeChannelTransferFixture::default()
+            .with_new_member_channel_owner(SECOND_MEMBER_ID)
+            .call_and_assert(Ok(()));
 
-        let curator_group_id = Content::next_curator_group_id();
+        CancelChannelTransferFixture::default()
+            .with_channel_id(ChannelId::from(2u32))
+            .call_and_assert(Err(Error::<Test>::ChannelDoesNotExist.into()))
+    })
+}
 
-        let price = 100;
-        UpdateChannelTransferStatusFixture::default()
-            .with_new_channel_owner(ChannelOwner::CuratorGroup(curator_group_id))
-            .with_price(price)
-            .call_and_assert(Ok(()));
+#[test]
+fn cancel_channel_transfer_fails_with_non_member_owner() {
+    with_default_mock_builder(|| {
+        ContentTest::with_member_channel().setup();
+        InitializeChannelTransferFixture::default().call_and_assert(Ok(()));
 
-        let member1_balance = Balances::<Test>::usable_balance(&DEFAULT_MEMBER_ACCOUNT_ID);
-        <Test as Config>::ContentWorkingGroup::set_budget(INITIAL_BALANCE);
+        CancelChannelTransferFixture::default()
+            .with_sender(THIRD_MEMBER_ACCOUNT_ID)
+            .with_actor(ContentActor::Member(THIRD_MEMBER_ID))
+            .call_and_assert(Err(Error::<Test>::ActorNotAuthorized.into()))
+    })
+}
 
-        AcceptChannelTransferFixture::default()
-            .with_origin(RawOrigin::Signed(LEAD_ACCOUNT_ID))
-            .with_price(price)
-            .call_and_assert(Ok(()));
+#[test]
+fn cancel_channel_transfer_fails_with_lead_and_member_channel() {
+    with_default_mock_builder(|| {
+        ContentTest::with_member_channel().setup();
+        InitializeChannelTransferFixture::default().call_and_assert(Ok(()));
 
-        assert_eq!(
-            <Test as Config>::ContentWorkingGroup::get_budget(),
-            INITIAL_BALANCE - price
-        );
-        assert_eq!(
-            Balances::<Test>::usable_balance(&DEFAULT_MEMBER_ACCOUNT_ID),
-            member1_balance + price
-        );
+        CancelChannelTransferFixture::default()
+            .with_sender(LEAD_ACCOUNT_ID)
+            .with_actor(ContentActor::Lead)
+            .call_and_assert(Err(Error::<Test>::ActorNotAuthorized.into()))
     })
 }
 
 #[test]
-fn update_transfer_status_blocked_during_upcoming_revenue_split() {
-    pub const SPLIT_STARTING_BLOCK: u64 = 10;
+fn cancel_channel_transfer_fails_with_collaborator_not_having_permissions() {
```

Comment:
Tests like:
- `cancel_channel_transfer_fails_with_collaborator_not_having_permissions`
- `cancel_channel_transfer_ok_with_member_owner`
- `cancel_channel_transfer_ok_with_lead`
- `cancel_channel_transfer_ok_with_collaborator_having_permissions`
- `cancel_channel_transfer_ok_with_curator_having_permissions`

are usually part of `tests/channel.rs`, because they're related to channel agent permissions and I thought it would be more clear to have all such tests in one place.

So, for example, we have:
```rust
///////////////////////////////////////////////////////////////////
/////// Channel agent / owner permissions - TransferChannel ///////
///////////////////////////////////////////////////////////////////

// Curator channels

fn unsuccessful_channel_transfer_by_curator_agent_without_permissions() { /* ... */ }
fn successful_transfer_by_curator_agent() { /* ... */ }
fn successful_channel_transfer_by_lead() { /* ... */ }

// Member channels

fn unsuccessful_channel_transfer_by_collaborator_without_permissions() { /* ... */ }
fn successful_channel_transfer_by_collaborator() { /* ... */ }
fn successful_channel_transfer_by_owner_member() { /* ... */ }
```

And I think you can add similar ones for `cancel_channel_transfer` there also (and remove them from here)

Reply:
I have added tests to the `tests/channel.rs`